.github/workflows/ci_tests_esp32.yml

```diff
@@ -0,0 +1,14 @@
+name: ESPRESSIF ESP32 CI Tests
+
+on: [pull_request]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    steps:
+    - uses: docker://espressif/idf:latest
+    - uses: actions/checkout@v2
+    - uses: snickerbockers/submodules-init@v4
+    - name: Build Esp32 Sample
+      run: docker run -v "${PWD}:/iot-middleware-freertos-samples" espressif/idf idf.py build -C /iot-middleware-freertos-samples/demos/projects/ESPRESSIF/esp32
```

Comment:
is it possible to add this build to following script?
https://github.com/Azure-Samples/iot-middleware-freertos-samples/blob/main/.github/scripts/ci_tests.sh

this way locally we can run this locally using this script.

Additionally, will docker command fail if the script passed to it fails?